pom.xml

```diff
@@ -1,272 +1,345 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="http://maven.apache.org/POM/4.0.0"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.bynder</groupId>
-  <artifactId>bynder-java-sdk</artifactId>
-  <version>2.2.7</version>
-  <packaging>jar</packaging>
+    <groupId>com.bynder</groupId>
+    <artifactId>bynder-java-sdk</artifactId>
+    <version>2.2.7</version>
+    <packaging>jar</packaging>
 
-  <name>Bynder Java SDK</name>
-  <description>The main goal of this SDK is to speed up the integration of Bynder customers who use
-    JAVA. Making it easier to connect to the Bynder API and executing requests on it.
-  </description>
-  <url>https://developer.bynder.com</url>
+    <name>Bynder Java SDK</name>
+    <description>The main goal of this SDK is to speed up the integration of Bynder customers who use
+        JAVA. Making it easier to connect to the Bynder API and executing requests on it.
+    </description>
+    <url>https://developer.bynder.com</url>
 
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
```

Comment:
I wonder why this was removed?

Reply:
I think this was overwritten when I was changing the deployment profile. I will add this back.